openpype/modules/ftrack/lib/avalon_sync.py

```diff
@@ -19,11 +19,8 @@
     CURRENT_PROJECT_SCHEMA,
     CURRENT_PROJECT_CONFIG_SCHEMA,
 )
-from openpype.api import (
-    Logger,
-    get_anatomy_settings
-)
-from openpype.lib import ApplicationManager
+from openpype.settings import get_anatomy_settings
+from openpype.lib import ApplicationManager, Logger
```

Comment:
Is this an intended refactor? (Just because it seems out of place in this PR and might be an oversight)

Reply:
Not related to PR but intended.